src/popup/components/Card.vue

```diff
@@ -70,21 +72,24 @@ export default {
     cardId: { type: String, default: null },
     icon: { type: Object, default: null },
   },
-  data: () => ({
-    CloseIcon,
-  }),
-  computed: {
-    ...mapState(['hiddenCards']),
-    styleComponent() {
-      return {
-        backgroundImage: (this.background)
-          ? `url("${this.background}")`
-          : null,
-      };
-    },
-    isVisible() {
-      return !this.cardId || !this.hiddenCards || !this.hiddenCards.includes(this.cardId);
-    },
+  setup(props) {
+    const hiddenCards = useState('hiddenCards');
+
+    const styleComponent = computed(() => ({
+      backgroundImage: props.background ? `url("${props.background}")` : null,
+    }));
+
+    const isVisible = computed(
+      () => !props.cardId
+      || !hiddenCards.value
+      || !hiddenCards.value.includes(props.cardId),
```

Comment:
1. For the multiline statements I suggest to put every statement in a separate line and wrap them with brackets.
2. Hidden cards are always an array, so there is no need to check if it exists. So this can be shortened.
3. I also suggest to inverse the logic.

```suggestion
      () => !(
        props.cardId
        && hiddenCards.value.includes(props.cardId)
      ),
```